scripts/sync-sdk-types.ts

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env tsx
+/**
+ * This script syncs types from payload.types.ts to SDKs
+ * Run this after `pnpm generate:types` to update SDK types
+ * 
+ * Note: Due to module augmentation issues, we manually define types
+ * instead of importing directly from payload.types.ts
+ */
+
+import { writeFileSync, readFileSync } from 'fs'
+import { join, dirname } from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
+
+const apisDoTypesPath = join(__dirname, '../sdks/apis.do/types.ts')
+
+const typesContent = readFileSync(apisDoTypesPath, 'utf8')
+
+const header = `/**
```

Comment:
why is header here if we're running on node and can load the types directly from /payload.types.ts?